lib/test/regression.js

```diff
@@ -65,61 +66,81 @@ describe('Messenger', () => {
         });
     });
 
-    it('Create Messenger', () => {
-        messenger = new Messenger(json.cookie, json.c_user, json.fb_dtsg);
-    });
+    describe('when data is loaded from the cookie', () => {
+        beforeEach(() => {
+            Crypt.load((err, data) => {
+                json = JSON.parse(data);
+            });
+        });
 
-    it('Send message', function(done) {
-    // Allow more time for network calls
-        this.timeout(5000);
+        it('Create Messenger', () => {
+            messenger = new Messenger(json.cookie, json.c_user, json.fb_dtsg);
+        });
 
-        messenger.sendMessage('ar.alexandre.rose', '731419306', 'Running tests - Send message', done);
-    });
+        it('getFriends', function(done) {
+            // Set a higher timeout for network calls
+            this.timeout(5000);
+            messenger.getFriends((_err, friends) => {
+                const friendIds = Object.keys(friends);
+                expect(friendIds.length).to.be.above(1);
+                done();
+            });
+        });
 
-    it('GetLastMessage', function(done) {
-    // Allow more time for network calls
-        this.timeout(5000);
+        it('Get threads', function (done) {
+            // Allow more time for network calls
+            this.timeout(5000);
 
-        messenger.getMessages('ar.alexandre.rose', '731419306', 10, (err, messages) => {
-            expect(messages.length).is.equal(10);
-            done();
+            messenger.getThreads((_err, messages) => {
+                expect(messages.length).to.be.above(1);
+                done();
+            });
         });
-    });
 
-    it('Get threads', function(done) {
-    // Allow more time for network calls
-        this.timeout(5000);
-
-        messenger.getThreads(true, done);
+        describe('interacting with friends', () => {
+            it('Send message', function (done) {
+                // Allow more time for network calls
+                this.timeout(5000);
+
+                messenger.getFriends((_, friends) => {
+                    const myself = friends[json.c_user];
+                    messenger.sendMessage(myself.vanity, myself.id, 'Running tests - Send message', done);
```

Comment:
This will ping the user who is using the library:

![image](https://user-images.githubusercontent.com/1649281/60829289-2cba4e00-a17a-11e9-8abc-ec7b7ded101e.png)